js/models/conversations.js

```diff
@@ -931,6 +931,13 @@
       }
       await this.setFriendRequestStatus(FriendRequestStatusEnum.requestSent);
     },
+    friendRequestTimerIsExpired() {
+      const unlockTimestamp = this.get('unlockTimestamp');
+      if (unlockTimestamp && unlockTimestamp - Date.now() > 0) {
```

Comment:
The most pedantic, minor-ific, just-to-have-the-final-word nitpick:
`if (unlockTimestamp && unlockTimestamp > Date.now()) {`
might read more organically.

Reply:
Yes good call